### ISSUE TITLE
Configuration for rule "@typescript-eslint/array-type" is invalid

### ISSUE THREAD

**Issue by Dulux-Oz** [origin-1/gulp-eslint-new#4]
Hi All,

Not sure if this error is actually in this package or in one of its dependencies (some of its dependencies have been updated in the last week or so, so this issue may disappear if this package was updated?) but as this is the package we're calling I thought we'd start here. :-)

Using latest versions of all packages (ie `npm -D update`).

Error:
~~~
c:\my_project\node_modules\@eslint\eslintrc\lib\shared\config-validator.js:143
                throw new Error(`${source}:\n\t${enhancedMessage}`);
                      ^
<ref *1> PluginError: .eslintrc.json:
        Configuration for rule "@typescript-eslint/array-type" is invalid:
        Value "array" should be object.
~~~

We're trying to lint a sample/simple ts file.

Thanks in advance
Dulux-Oz

**Comment by fasttime** [comment]
This sounds like a configuration problem. Have you checked if your `.eslintrc.json` file contains a definition for `@typescript-eslint/array-type`?

**Comment by Dulux-Oz** [comment]
@fasttime So, that's what I thought originally, but I went looking for `@typescript-eslint/array-type` in our `.eslintrc.json` file and couldn't find it. Hence the raised issue.

But with your confirmation/suggestion that it *was* in the `.eslintrc.json` file I went looking again (actually went line-by-line this time, as opposed to a global-search) - and this time I found it and have now corrected it (I must have miss-spelt it in my original global-search).

So, yes, my goof, not an error in the package - sorry about that! And thanks for the help.

Cheers
Dulux-Oz
